workshops/templates/workshops/all_airports.html

```diff
@@ -9,7 +9,11 @@
 {% endblock %}
 
 {% block content-fluid %}
+    {% if perms.workshops.add_airport %}
     <p><a href="{% url 'airport_add' %}" class="btn btn-success">New airport</a></p>
+    {% else %}
+    <p><a href="{% url 'airport_add' %}" class="btn btn-success disabled">New airport</a></p>
+    {% endif %}
```

Comment:
Same as above (move conditional inline to eliminate duplication) - I'll skip duplicating this comment below :-)